ivy/array/extensions/layers.py

```diff
@@ -262,3 +262,45 @@ def pad(
             out=out,
             **kwargs,
         )
+
+    def rfft(
+            self: ivy.Array,
+            n: Optional[int] = None,
+            norm: Optional[str] = None,
+            /,
+            *,
+            out: Optional[ivy.Array] = None
+    ) -> ivy.Array:
+        """
+        ivy.Array instance method variant of ivy.rfft. This method simply
+        wraps the function, and so the docstring for ivy.rfft also applies to
+        this method with minimal changes.
+
+        Parameters
+        ----------
+        self
+            The real input array.
+        n
+            Signal length. If given, the input will either be zero-padded or trimmed
+            to this length before computing the real FFT.
+        norm
+            Normalization mode.
+        out
+            optional output array, for writing the result to.
+
+        Returns
+        -------
+        ret
+            The output array.
+
+        Examples
+        --------
+        >>> x = ivy.array([0, 1, 2, 3])
+        >>> ivy.rfft(x)
```

Comment:
Also, please refer to the docs to see the correct way of calling array instance methods!